Description: Upstream fix for debian bug 736374
Don't eat mouse wheel events if we can't handle them in wxGTK.

The recent changes to mouse wheel scrolling broke it for the generic
wxListCtrl and probably any other situation in which scrollbars don't scroll
the window itself but some subwindow of it. In this case, the scroll wheel
event handler doesn't have any scrollbar to adjust, so it simply does nothing
and shouldn't return true, as it did since r74911.

Just return false in case we are not doing anything to let the mouse wheel
work as before.


---
Origin: upstream, http://trac.wxwidgets.org/changeset/75191
Bug-Debian: http://bugs.debian.org/736374
Forwarded: not-needed
Last-Update: <2014-04-28>

--- wxwidgets3.0-3.0.0.orig/src/gtk/window.cpp
+++ wxwidgets3.0-3.0.0/src/gtk/window.cpp
@@ -1723,6 +1723,9 @@ scroll_event(GtkWidget* widget, GdkEvent
 
     if (!win->GTKProcessEvent(event))
     {
+        if (!range)
+            return false;
+
         if (direction == GDK_SCROLL_UP || direction == GDK_SCROLL_LEFT)
             step = -step;
         AdjustRangeValue(range, step);
